backend/src/services/model.ts

```diff
@@ -268,6 +279,9 @@ export type UpdateModelParams = Pick<
 >
 export async function updateModel(user: UserInterface, modelId: string, diff: Partial<UpdateModelParams>) {
   const model = await getModelById(user, modelId)
+  if (model.settings.mirror.sourceModelId !== '' || model.settings.mirror.destinationModelId !== '') {
```

Comment:
You should be able to update a model regardless of whether it is a mirrored model or it is going to be copied to a mirrored model. Instead, you stop any update that is trying to set the source model ID (trying to change an existing model to be a mirrored model) and stop a destination model ID being set if the source model ID is already set (stop a mirrored model being mirrored somewhere else)